core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -71,13 +82,18 @@ public String getPulsarImage() {
     }
 
     public boolean isDisableChecks() {
-        return Boolean.parseBoolean((String) properties.getOrDefault("checks.disable", "false"));
+        return Boolean.parseBoolean((String) environmentProperties.getOrDefault("checks.disable", "false"));
```

Comment:
was never supposed to be disabled in classpath' file, but some users were misusing it and setting it "for everyone".

Reply:
Seems sensible 👍